tests/cxx/unit_tests/xyz_to_mol.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2024 NWChemEx-Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "../test_nux.hpp"
+#include <nux/nux.hpp>
+
+DECLARE_MODULE(PseudoZFromSymbol);
+inline MODULE_CTOR(PseudoZFromSymbol) {
+  satisfies_property_type<simde::ZFromSymbol>();
+}
+inline MODULE_RUN(PseudoZFromSymbol){
+  simde::type::atomic_number Z = 1;
+  auto rv = results();
+  return simde::ZFromSymbol::wrap_results(rv, Z);
+}
+
+DECLARE_MODULE(PseudoAtomFromZ);
+inline MODULE_CTOR(PseudoAtomFromZ) {
+  satisfies_property_type<simde::AtomFromZ>();
+}
+inline MODULE_RUN(PseudoAtomFromZ) {
+  const auto& [Z] = simde::AtomFromZ::unwrap_inputs(inputs);
+  simde::type::atom atom{"H", Z, 1837.4260218693814, 0.0, 0.0, 0.0};
+  auto rv = results();
+  return simde::AtomFromZ::wrap_results(rv, atom);
+}
```

Comment:
This is probably simpler and cleaner with [facade](https://nwchemex.github.io/PluginPlay/pluginplay_cxx_api/classpluginplay_1_1detail___1_1FacadeModule.html) or [lambda](https://nwchemex.github.io/PluginPlay/pluginplay_cxx_api/classpluginplay_1_1detail___1_1LambdaModule.html) modules.